Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM phusion/baseimage:0.11
+
+CMD ["/sbin/my_init"]
+
+RUN apt-get update && apt-get -y upgrade && apt-get -y install \
+    nano \
+    rsync \
+    software-properties-common \
+    sudo \
+    wget && \
+    apt-get purge -y syslog-ng-core && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+ARG DOKKU_TAG=0.17.6
+ARG DOKKU_GID=200
+ARG DOKKU_UID=200
+ARG DOKKU_DOCKERFILE=true
+ARG DOKKU_WEB_CONFIG=false
+ARG DOKKU_HOSTNAME=dokku.invalid
+ARG DOKKU_VHOST_ENABLE=false
+ARG DOKKU_SKIP_KEY_FILE=true
+
+RUN addgroup --gid $DOKKU_GID dokku && \
+    adduser --uid $DOKKU_UID --gid $DOKKU_GID --disabled-password --gecos "" "dokku"
+
+RUN mkdir /etc/nginx/
+COPY tests/dhparam.pem /etc/nginx/dhparam.pem
```

Comment:
Why does this live in the `tests` subdir? Looks a bit out of place to me if it’s intended to be used in a production build.

Reply:
Removed, it wasn't actually necessary.